common/utils.go

```diff
@@ -40,6 +41,13 @@ func FormatValue(val any) any {
 			return parsed.UTC().Format(time.RFC3339)
 		}
 		return s
+	case uint64:
+		if val <= math.MaxInt64 {
+			return int64(val)
+		}
+
+		// this will make avro encoding fail, as it doesn't support uint64.
+		return val
```

Comment:
But this returns a schema, not a value, right?

Reply:
Support for uint64 in conduit afaik is not there yet. What you suggested @nickchomey doesn't really work, as every single destination connector would have to understand the format you wrote. I'm not a conduit developer, just a connector developer, so I dont' really know the internals / what it is supposed to happen in order to support uint64. I guess it is more of agreeing with a standard.

The patch I wrote should work for uint64 that are less than the max int64. Do you have uint64 larger than that?

Reply:
What we're doing here is already a workaround that makes it work at least for the smaller uints. It's already not ideal, because we lose the type information, the destination will think the value is a `int64`, but hey, it's a quick fix that solves your problem 🤷 The real fix would be to convince the Avro lib to encode the uint64 correctly, but that's easier said than done, because we are working with `interface{}` types, we have no idea about the type in advance.

So with this fix any uint64 smaller than max int64 will be transmitted as int64, any other will result in the same error as now.